C-Plus-Plus/cryptography/Modified_Caeser_Cipher.cpp

```diff
@@ -0,0 +1,56 @@
+//Modified Caeser cipher
+
+/*
+It is a type of susbtitution cipher which encrypts the plain text 
+message into a cipher text by simply replacing each letter
+of a given text by some places down the alphabet.
+
+
+Example:
+Text : TESSERACTCODING
+Shift: 4
+Cipher: XIWWIVEGXGSHMRK
+
+Text : NEOALGO
+Shift: 2
+Cipher: PGQCNIQ
+
+Time Complexity: O(n)
+Space Complexity: O(n)
+
+Optimised method:
+Change the plain text itself instead of initializing a new
+string varibale if plain text is mutable.
+Space complexity: O(1)
+Time Complexity: O(n)
+
+*/
+
+
+#include<bits/stdc++.h>
+using namespace std;
+
+int main(){
+	
+	string plain,encrypt;
+    cout<<"Enter the plain text: ";
+	cin>>plain;
+	
+	int shift;
+    cout<<"Enter the integer key to shift: ";
+	cin>>shift;
+	
+	encrypt="";
```

Comment:
Push all the functional Code outside of main() to a reusable function. The main() function strictly serves our purpose of driver code.